beyond-all-reason/Beyond-All-Reason
# feat!(group_recall): Remove group recall widget

Previously present to address an issue with engine not having a timer for focusing groups with double tap, this widget is not necessary anymore with group select and group focus actions

### Work done

Removed group recall fix widget, used decoupled select and focus actions in default group hotkeys.

#### Setup

- `/cheat`
- `/give 50 armpw`
- Split into two groups (1, 2) and move them a bit apart


#### Test steps
- Tap 1, check it selects group 1
- Tap 2, check it selects group 2
- Double tap 1, check it focuses camera on group 1
- Double tap 2, check it focuses camera on group 2
- Slow double tap, check it does not focus camera
